vimrc

```diff
@@ -46,12 +50,14 @@ augroup vimrcEx
   autocmd BufRead,BufNewFile .{jscs,jshint,eslint}rc set filetype=json
 
   " ALE linting events
-  set updatetime=1000
-  let g:ale_lint_on_text_changed = 0
-  autocmd CursorHold * call ale#Lint()
-  autocmd CursorHoldI * call ale#Lint()
-  autocmd InsertEnter * call ale#Lint()
-  autocmd InsertLeave * call ale#Lint()
+  if v:version >= 800
```

Comment:
My local (recent install) of neovim returns 704, so I think the `has('nvim')` addition makes sense. Since the conditional is a bit more complex now, what do you think of introducing a `function! s:has_async()` in the vimrc. I expect that the function will be available in the `vimrc.bundles` since it is sourced from the vimrc, although you might need to make it global rather than `s:` script local.